site/en/blog/has-with-cq-m105/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "@container and :has(): two powerful new responsive APIs landing in Chromium 105"
+description: >
+  "Container queries and :has() are a match made in responsive heaven. Luckily, both of these features are landing together in Chromium 105. This is a huge release with two highly-requested features for responsive interfaces!"
+layout: 'layouts/blog-post.njk'
+date: 2022-07-25
+hero: 'image/HodOHWjMnbNw56hvNASHWSgZyAf2/KBYS0gF3R7mdhjVBwvbn.jpg'
+alt: >
+  Image of a rocketship from SpaceX.
+tags:
+  - chrome-105
+  - css
+authors:
+  - unakravets
+---
+
+Container queries and :has() are a match made in responsive heaven. Luckily, both of these features are landing together in Chromium 105. This is a huge release with two highly-requested features for responsive interfaces!
+
+## Container Queries: a quick summary
+
+Container queries enable developers to query a parent selector for its size and styling information, making it possible for a child element to own its responsive styling logic, no matter where it lives on a web page.
+
+Instead of relying on the viewport for styling input such as available space, developers now have the ability to query the size of in-page elements too.. This capability means that a component owns its responsive styling logic. This makes the component much more resilient, as the styling logic is attached to it, no matter where it appears on the page.
```

Comment:
```suggestion
Instead of relying on the viewport for styling input such as available space, developers now have the ability to query the size of in-page elements too. This capability means that a component owns its responsive styling logic. This makes the component much more resilient, as the styling logic is attached to it, no matter where it appears on the page.
```